pkg/drivers/kic/oci/volumes.go

```diff
@@ -30,6 +31,47 @@ import (
 	"k8s.io/klog/v2"
 )
 
+// RemoveVolume removes a volume
+func RemoveVolume(ociBin string, name string) error {
```

Comment:
this func is not used anywhere ?

Reply:
For some reason, the "delete" command (above) is talking directly to oci module. And the usual Delete() is never called ?

oci.RemoveVolume
oci.DeleteAllVolumesByLabel

oci.RemoveNetwork
oci.DeleteKICNetworks

Reply:
The naming of those functions is not great, and it seems like the network deletion could do with a specific label ?